### PR TITLE
Add suggestion about python versions support

### PR DESCRIPTION
**Pull request by xmnlab**
Not sure where would be the best place to add this information. 
this PR is related to https://github.com/pyOpenSci/cookiecutter-pyopensci/issues/17

any suggestion or recommendation here would be very welcome :)

### CODE REVIEW

File: authoring/overview.md [hunk 1]
```diff
@@ -99,6 +99,9 @@ pyOpenSci projects should use an open source software license that is approved b
 - **Better/Best:** Choose a license based on your needs and future use of package, plus explain your choice in your submission for review.
 
 ## Other recommendations
+### Python version support
+As keeping compatibility with old Python versions could be a hard task, pyOpenSci suggests that the package should supports, at least, latest three Python versions, as much as possible.
```
Comment (choldgraf) [inline authoring/overview.md:103]:
```suggestion
You should always be explicit about which versions of Python your package supports.
Keeping compatibility with old Python versions can be difficult as functionality changes.
A good rule of thumb is that the package should support, at least,
the latest three Python versions (e.g., 3.8, 3.7, 3.6).
```

Comment (choldgraf) [reply authoring/overview.md:103]: how about this?

Comment (xmnlab) [reply authoring/overview.md]: sounds great! suggestion applied! thank you so much!

**Comment by choldgraf** [comment]
thanks for the improvement!!!

**Comment by lwasser** [comment]
Thank you both!!

**Pull request merged**